test/manage/manage-bar.test.ts

```diff
@@ -45,6 +45,13 @@ describe('Manage bar', () => {
     expect(el.shadowRoot?.querySelector('.unselect-all-btn')).to.exist;
   });
 
+  it('have delete button disabled', async () => {
+    const el = await fixture<ManageBar>(
+      html`<manage-bar ?enableRemoveButton=${false}></manage-bar>`
```

Comment:
Note `?myattr=${false}` is a no-op in most contexts (it just means "don't include a `myattr` attribute on the element"). But I suppose it's okay here just as an extra visual indication of what's being tested.